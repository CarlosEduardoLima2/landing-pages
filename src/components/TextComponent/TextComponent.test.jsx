import { screen } from '@testing-library/react';
import { TextComponent } from '.';
import { renderTheme } from '../../styles/render-theme';

describe('<TextComponent />', () => {
  it('should render a text', () => {
    renderTheme(<TextComponent>TextComponent</TextComponent>);
    expect(screen.getByText('TextComponent')).toBeInTheDocument();
  });

  it('should match snapshot', () => {
    const { container } = renderTheme(<TextComponent>TextComponent</TextComponent>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
