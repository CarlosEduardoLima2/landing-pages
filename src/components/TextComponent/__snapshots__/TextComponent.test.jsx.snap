// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextComponent /> should match snapshot 1`] = `
.c0 {
  font-size: 2.4rem;
}

<p
  class="c0"
>
  TextComponent
</p>
`;

exports[`<TextComponent /> should render a text 1`] = `
.c0 {
  font-size: 2.4rem;
}

<p
  class="c0"
>
  TextComponent
</p>
`;

exports[`<TextComponent /> to match snapshot 1`] = `
.c0 {
  font-size: 2.4rem;
}

<p
  class="c0"
>
  TextComponent
</p>
`;
